refl1d/reflectivity.py

```diff
@@ -452,63 +772,67 @@ def convolve_sampled(xi, yi, xp, yp, x, dx):
     return y
 
 def test_uniform():
-    xi = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-    yi = [1, 3, 1, 2, 7, 3, 1, 2, 1, 3]
-    _check_uniform("uniform aligned", xi, yi, x=[2, 4, 6, 8], dx=1)
-    _check_uniform("uniform unaligned", xi, yi, x=[2.5, 4.5, 6.5, 8.5], dx=1)
-    _check_uniform("uniform wide", xi, yi, x=[2.5, 4.5, 6.5, 8.5], dx=3)
+    xi=[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    yi=[1, 3, 1, 2, 7, 3, 1, 2, 1, 3]
+    _check_uniform("uniform aligned", xi, yi, x = [2, 4, 6, 8], dx = 1)
+    _check_uniform("uniform unaligned", xi, yi,
+                   x = [2.5, 4.5, 6.5, 8.5], dx = 1)
+    _check_uniform("uniform wide", xi, yi, x = [2.5, 4.5, 6.5, 8.5], dx = 3)
     # Check bad values
-    ystar = convolve(xi, yi, x=[-3, 13], dx=[1/np.sqrt(3)]*2, resolution='uniform')
+    ystar = convolve(xi, yi, x = [-3, 13], dx = [1/np.sqrt(3)]*2, resolution ='uniform')
     assert (ystar == 0.).all()
 
-    xi = [1.1, 2.3, 2.8, 4.2, 5, 6, 7, 8, 9, 10]
-    yi = [1, 3, 1, 2, 7, 3, 1, 2, 1, 3]
-    _check_uniform("uniform unaligned", xi, yi, x=[2.5, 4.5, 6.5, 8.5], dx=1)
+    xi=[1.1, 2.3, 2.8, 4.2, 5, 6, 7, 8, 9, 10]
+    yi=[1, 3, 1, 2, 7, 3, 1, 2, 1, 3]
+    _check_uniform("uniform unaligned", xi, yi,
+                   x = [2.5, 4.5, 6.5, 8.5], dx = 1)
 
 def _check_uniform(name, xi, yi, x, dx):
     # Note: using fixed dx since that's all _check_spline supports.
-    ystar = convolve(xi, yi, x, [dx/np.sqrt(3)]*len(x), resolution='uniform')
-    #print("xi", xi)
-    #print("yi", yi)
-    #print("x", x, "dx", dx)
-    #print("ystar", ystar)
-    xp = [-dx, -dx, dx, dx]
-    yp = [0, 0.5/dx, 0.5/dx, 0]
+    ystar=convolve(xi, yi, x, [dx/np.sqrt(3)]*len(x), resolution = 'uniform')
```

Comment:
PEP8: Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not).